velox/substrait/tests/JsonToProtoConverter.cpp

```diff
@@ -33,5 +33,10 @@ void JsonToProtoConverter::readFromFile(
   VELOX_CHECK(
       status.ok(),
       "Failed to parse Substrait JSON: {}",
-      status.message().ToString());
+#if PROTOBUF_VERSION < 4022000
```

Comment:
Can we use `std::string_view(status.message.data(), static_cast<size_type>(status.message.size()))` and avoid this ifdef? The above API is available in both `absl::string_view` and older protobuf `stringpiece`.

Reply:
Fixed!